frontend/public/components/_overview.scss

```diff
@@ -26,8 +26,72 @@
 }
 
 .overview {
-  width: 100%;
+  bottom: 0;
   display: flex;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 67px;
+  @media(min-width: $screen-sm-min) {
+    left: 220px;
+    top: 100px;
+  }
+
+  .overview__middle {
+    height: 100%;
+    overflow: hidden;
+    overflow-y: auto;
+    width: 100%;
+    -webkit-overflow-scrolling: touch;
+
+    .overview__middle-section {
+      flex-grow: 1;
+    }
+  }
+
+  .overview__sidebar--right {
+    background: #fff;
+    bottom: 0;
+    box-shadow: -5px 0 3px 2px rgba(0,0,0,0.05), -1px 0px 3px 2px rgba(0,0,0,0.2);
+    display: none;
+    position: fixed;
+    right: 0;
+    top: 100px;
+    width: 43%;
+    z-index: 5;
+    @media(max-width: $screen-xs-max) {
+      top: 67px;
```

Comment:
Should use a var.

Global notifications are currently not in use (impersonate user is currently disabled), but this will be problematic if they are re-enabled.